pkgs/by-name/ve/versionCheckHook/hook.sh

```diff
@@ -5,6 +5,7 @@ _handleCmdOutput(){
         --chdir=/ \
         --argv0="$(basename "${command[0]}")" \
         $( [[ -z "$3" || "$3" = "0" ]] && echo --ignore-environment ) \
+        $(for var in $4; do echo "$var=$(eval "echo -n \"\$$var\"")"; done) \
```

Comment:
Can we do bash native variable indirection here with `${!var}` ?

Reply:
Thanks, thats exactly what I was looking for!